src/test/e2e/e2e_zarf_create_test.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2eZarfCreate(t *testing.T) {
+	defer e2e.cleanupAfterTest(t)
```

Comment:
This is failing on the k3s portion of the e2e test because the cleanup function is trying to do a `zarf destroy ...` when there is no k8s cluster running (since Zarf would spawn that when running the init command) which leads to some exception being thrown. 

I propose adding a check before it runs `zarf destroy`, something like this perhaps:
```
func (e2e *ZarfE2ETest) cleanupAfterTest(t *testing.T) {
	// Check if we have a running cluster to
	err := clusters.TryValidateClusterIsRunning()
	if err == nil {
		// Use Zarf to perform chart uninstallation
		output, err := e2e.execZarfCommand("destroy", "--confirm", "--remove-components", "-l=trace")
		require.NoError(t, err, output)
	}

    ....
```

Thoughts?

Reply:
👍

Reply:
Added this with latest commit. Good catch, I had no idea why the k3s test kept failing